volatility3/cli/text_renderer.py

```diff
@@ -131,12 +131,19 @@ def display_disassembly(disasm: interfaces.renderers.Disassembly) -> str:
 
 
 class CLIRenderer(interfaces.renderers.Renderer):
-    """Class to add specific requirements for CLI renderers."""
+    """Class to add specific requirements for CLI renderers.
+    
+    Args:
+        output_filename (optionnal): The output file descriptor to save the output of a module inside 
+    """
 
     name = "unnamed"
     structured_output = False
     filter: text_filter.CLIFilter = None
 
+    def __init__(self, outfd):
+        self.outfd = outfd
+
```

Comment:
This doesn't inherit from the renderer's `__init__` (https://github.com/nathan-out/volatility3-output-module/blob/ffaa7738329da6e21f4ae3505576dc003f05ae0f/volatility3/framework/interfaces/renderers.py#L38) so will break inheritance.  As mentioned we've never fleshed out what the render options are, but the idea was to allow renderers to accept options and parameters as they needed, whereas this would be rushing the process for a single feature that hasn't yet been demonstrated to be necessary?